unison-core/src/Unison/NamesWithHistory.hs

```diff
@@ -250,33 +249,6 @@ termName length r NamesWithHistory {..} =
     hq n = HQ'.take length (HQ'.fromNamedReferent n r)
     isConflicted n = R.manyDom n (Names.terms currentNames)
 
-suffixedTypeName :: Int -> Reference -> NamesWithHistory -> [HQ'.HashQualified Name]
```

Comment:
This was deconstructed and moved to the `PrettyPrinterEnv.Names` module so we can share a prioritization scheme across both suffixified and unsuffixified versions.